src/forecast.jl

```diff
@@ -82,12 +82,44 @@ error(real::Vector{T}, forecast::Vector{T}) where {T} = real .- forecast
 absolute_percentage_error(real::Vector{T}, forecast::Vector{T}) where {T} =
     abs.(error(real, forecast) ./ real)
 
+"""
+    cross_validation(fit_input::FitInput{T}, fit_function::Function, predict_function::Function, metric_function::Function,
+                            min_history::Int, horizon::Int)
+
+Function that receives all avaiable data in fit_input, a fit_function and a prediction_funtion,
+a validation metric (eg. mape), a min_history as a starting point and a prediction horizon.
+Returns the result of the cross validation separeted by lead time.
+"""
+function cross_validation(fit_input::FitInput{T},
+                            fit_function::Function, 
+                            predict_function::Function,
+                            metric_function::Function,
```

Comment:
you should use this function `forecast_metrics`

Reply:
and then make another function to aggregate the vector of metrics, maybe then add this metric function